css/style.css

```diff
@@ -360,12 +360,17 @@ nav.navbar.navbar-default {
 	margin: 2em 0;
 }
 
-.entry-meta {
-	margin: 1em 0;
-	font-size: 14px;
+.article-head p.entry-meta,
+.non-link a {
+	font-size: 12px;
 	color: #777;
 }
 
+.non-link a {
```

Comment:
yes, [this page's](https://libraries-dev.mit.edu/akdc/film-screening-5-broken-cameras/) code looks good now. Can you remove the .non-link rule as well?